service/src/test/java/uk/gov/hmcts/reform/fpl/service/OrganisationServiceTest.java

```diff
@@ -67,7 +76,7 @@
     @BeforeEach
     void setup() {
         when(authTokenGenerator.generate()).thenReturn(SERVICE_AUTH_TOKEN);
-        when(requestData.authorisation()).thenReturn(AUTH_TOKEN);
+        lenient().when(requestData.authorisation()).thenReturn(AUTH_TOKEN);
```

Comment:
you can do instead:

@Mock(lenient = true)
private RequestData requestData;

which is preferable as test code should focus on test scenario and not of technical aspects (should be as close to plain english as possible)

alternatively you can add at the class level:
@MockitoSettings(strictness = Strictness.LENIENT)